README.md

```diff
@@ -1,568 +1,181 @@
-
-All that’s needed to use a model with AIConfig is a ModelParser that knows
-
-- how to serialize data from a model into the aiconfig format
-- how to deserialize data from an aiconfig into the type the model expects
-- how to run inference for model.
-
-For more details, see [Extensibility](https://aiconfig.lastmileai.dev/docs/extensibility).
-
-### When should I store outputs in an `aiconfig`?
-
-The `AIConfigRuntime` object is used to interact with an aiconfig programmatically (see [SDK usage guide](#aiconfig-sdk)). As you run prompts, this object keeps track of the outputs returned from the model.
-
-You can choose to serialize these outputs back into the `aiconfig` by using the `config.save(include_outputs=True)` API. This can be useful for preserving context -- think of it like session state.
+## Why is this important?
 
-For example, you can use aiconfig to create a chatbot, and use the same format to save the chat history so it can be resumed for the next session.
+Today application code is tightly coupled with the gen AI settings for the application -- prompts, parameters, and model-specific logic is entangled in app code.
 
-You can also choose to save outputs to a _different_ file than the original config -- `config.save("history.aiconfig.json", include_outputs=True)`.
+This makes incredibly difficult to iterate on and deploy a production-grade generative AI application. Here are common issues today:
 
-### Why should I use `aiconfig` instead of things like [configurator](https://pypi.org/project/configurator/)?
+- increased complexity to iterate on prompts and model parameters
+- dificulty in switching between models across providers
+- limited collaboration with prompt engineers, who might be domain experts and not developers
+- limited evaluation capabilities for prompt and model performance
 
-It helps to have a [standardized format](http://aiconfig.lastmileai.dev/docs/overview/ai-config-format) specifically for storing generative AI prompts, inference results, model parameters and arbitrary metadata, as opposed to a general-purpose configuration schema.
+AIConfig helps unwind complexity by separating prompts, model parameters, and model-specific logic from your application.
 
-With that standardization, you just need a layer that knows how to serialize/deserialize from that format into whatever the inference endpoints require.
+- simplifies application code -- simply call `config.run()`.
+- open the `aiconfig` in a playground to iterate quickly. prompt engineers can start here and handoff the `aiconfig` to developers.
+- version control and evaluate the `aiconfig` - it's the AI artifact for your application.
 
-### This looks similar to `ipynb` for Jupyter notebooks
+<div style="text-align:center;">
+    <img src="aiconfig-docs/static/img/aiconfig_dataflow.png" alt="AIConfig flow" style="width:40%;">
+</div>
```

Comment:
I think having this image closer to the start of the readme will be really helpful